ui/modal/modalImageUpload/view.jsx

```diff
@@ -14,10 +14,13 @@ type Props = {
 
 function ModalImageUpload(props: Props) {
   const { closeModal, currentValue, title, assetName, helpText, onUpdate } = props;
+  const filters = React.useMemo(() => [{ name: 'Images', extensions: ['jpg', 'png', 'gif'] }]);
```

Comment:
this blocks `jpeg` - possibly some other valid types. svg has been possible and allowed so far.